### PR TITLE
Fix ssh

### PR DESCRIPTION
**Pull request by doodles526**
## Pull Request Checklist

**Is this in reference to an existing issue?**
Yes, https://github.com/golang/go/issues/19767
#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Existing tests pass

#### Purpose

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 
 ### Fixed
+* Complies with bug introduces from https://github.com/golang/go/issues/19767
```
Comment (mbyczkowski) [inline CHANGELOG.md]: `Complies with ssh behavior change in ...` maybe?

File: local/ssh_handler.go [hunk 2]
```diff
@@ -118,7 +118,8 @@ func (s *SSHHandler) dial() (*ssh.Client, net.Listener, error) {
 		Auth: []ssh.AuthMethod{
 			ssh.Password(s.FlyToken),
 		},
-		Timeout: sshConnTimeout,
+		Timeout:         sshConnTimeout,
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```
Comment (mbyczkowski) [inline local/ssh_handler.go]: that should do for now 👍

File: CHANGELOG.md (hunk 1 above)
Comment (doodles526) [reply CHANGELOG.md]: Hmm. Yeah I agree. Will change

Comment (doodles526) [reply CHANGELOG.md]: Fixed here 132aa9b

Comment (doodles526) [reply CHANGELOG.md]: Err b7c3acd

**Comment by doodles526** [comment]
@mbyczkowski Back at ya! Should be ready to go

**Pull request merged**